Components/Lists/lists.css

```diff
@@ -0,0 +1,162 @@
+.list-container
+{
+    width: 100%;
+    height: fit-content;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    background-color: #F5F7F9;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: 4.8rem 1.6rem;
+}
+
+.spaced-ul
+{
+    max-width: 100%;
+    list-style: none;
+    margin: 0;
+    padding: 0 3rem;
+    border-radius: 10px;
+    background-color: var(--background-color);
+}
+
+.spaced-ul li
+{
+    margin: 5rem 0;
+    border: 1px solid var(--primary-dark-blue-color);
+    border-radius: 10px;
+    background-color: var(--background-color);
+}
+
+.spaced-post
+{
+    max-width: 100%;
+    padding: 1rem 0;
+    border-radius: 10px;
+}
+
+.spaced-post hr
+{
+    margin: 0;
+}
+
+.post-header
+{
+    height: 4.8rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.post-header .icon-btn
+{
+    min-width: 3.2rem;
+    min-height: 3.2rem;
+    height: fit-content;
+    padding: 0;
+    margin: 0 1rem;
+}
+
+.post-header-left-container, .active-connections-header-container
+{
+    width: fit-content;
+    max-width: 100%;
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
+}
+
+.post-header-left-container h2
+{
+    margin: 0 2rem;
+}
+
+.post-header-right-container
+{
+    width: fit-content;
+    display: flex;
+}
+
+.post-caption
+{
+    padding: 1rem 1rem;
+}
+
+.post-img
+{
+    max-width: 100%;
+}
+
+.post-stats
+{
+    display: flex;
+    padding: 0 5px;
+    justify-content: space-between;
+}
+
+.post-stats-left-container
+{
+    display: flex;
+    align-items: center;
+    padding: 0 1rem;
```

Comment:
padding is a little less from my perspective, and you should just keep two buttons there 3 with same significance confuses the user

Reply:
Thanks, will look into it.